src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,26 @@
+package seedu.addressbook.data.person;
+
+/**
+ * Represents a printable object
+ *
+ */
+public interface Printable {
+    /**
+     * Returns printable message
+     * @return
+     */
+    public String getPrintableString();
+
+    /**
+     * Returns printable message combined from other Printable instances
+     * @param printables
+     * @return
+     */
+    default String getPrintableString(Printable... printables){
```

Comment:
I'm thinking there might be other classes that implements Printable and the function of printing a collection of Printables is not specific to the Person class